.github/workflows/helm-enterprise.yml

```diff
@@ -123,24 +128,35 @@ jobs:
         run: >-
           helm dep up ./helm/alfresco-content-services &&
```

Comment:
you can move this to a dedicated step, but more importantly I see only now that it should be `helm dep build` and not `up` 😱  check also for other usages

Reply:
will be tackled in OPSEXP-2590